mk/install-build-tools.sh

```diff
@@ -91,6 +91,12 @@ case $target in
     libc6-dev-mipsel-cross \
     qemu-user
   ;;
+--target=riscv64gc-unknown-linux-gnu)
+  install_packages \
+    gcc-riscv64-linux-gnu \
+    libc6-dev-riscv64-cross \
```

Comment:
Does this need to be changed with the switch to clang? Look at the other `use_clang` targets.

Reply:
Yes, indeed, I did check if `gcc-riscv64-linux-gnu` and `libc6-dev-riscv64-cross` were still applicable but missed the `use_clang` setting. Adding that

Reply:
Added in f2ad112